app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
```

Comment:
I would suggest using a bit more semantic HTML in your views rather than wrapping everything in divs.  However that's a minor criticism.